### ISSUE TITLE
No event before slide changes only

### ISSUE THREAD

**Issue by HTMeLwyn** [metafizzy/flickity#593]
So I noticed there are the „select“ and „scroll“ events.
However, they both also trigger when the slideshow/browser is resized (see: https://codepen.io/anon/pen/QgVpjz?editors=1111).
Is there a possibility to add an event listener before a slide is about to change? (be it by drag, navigation or autoplay).

That would really help me out.
Cheers!

**Comment by desandro** [comment]
Thanks for reporting this issue. Can you specify what you mean by "before a slide is about to change"? Do you mean change in size, or where the slider is positioned?

See also https://github.com/metafizzy/flickity/issues/586#issuecomment-313395681

**Comment by HTMeLwyn** [comment]
What I meant by this is that I need an event that fires when the current slide is switched.
Either by the user starting to drag/clicking on the navigation or by autoplay starting the transition to the next/previous slide.
I tried the solution you proposed in the comment you linked, but I came to the issue I just mentioned. Both events (select and scroll) also fire when the slider is resized. 
The codepen I linked to should make the issue more clear. When you resize the browser, both events fire.
